Uplift/Views/Cells/FacilitiesDropdownCell.swift

```diff
@@ -9,136 +9,96 @@ import UIKit
 
 class FacilitiesDropdownCell: UICollectionViewCell {
 
-    private enum FacilityName: String {
-        case bowlingLanes = "BOWLING LANES"
-        case equipment = "EQUIPMENT"
-        case fitnessCenter = "FITNESS CENTER"
-        case gymnasium = "GYMNASIUM"
-        case swimmingPool = "SWIMMING POOL"
-    }
-
-    private let headerImageView = UIImageView()
-    private let headerNameLabel = UILabel()
-    private let headerOpenLabel = UILabel()
-    private let headerView = DropdownHeaderView(frame: .zero, arrowImage: nil, arrowImageTrailingOffset: 0)
-
+    private let headerView = FacilitiesDropdownHeaderView(frame: .zero)
     private var collectionView: UICollectionView!
-    private var collectionViewHeight: CGFloat = 0
     private var dropdownView: DropdownView!
     private var facility: Facility!
-    private var height: CGFloat = 0
+    static let headerViewHeight: CGFloat = 52
+    static let collectionViewSpacing: CGFloat = 16
+    private var facilitiesIndex: Int!
+    private var headerViewTapped: ((Int) -> ())?
+
+    weak var delegate: GymDetailViewController!
 
     override init(frame: CGRect) {
         super.init(frame: frame)
 
+        contentView.backgroundColor = .white
+
         let collectionViewLayout = UICollectionViewFlowLayout()
-        collectionViewLayout.minimumInteritemSpacing = 16
+        collectionViewLayout.minimumInteritemSpacing = FacilitiesDropdownCell.collectionViewSpacing
+        collectionViewLayout.sectionInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
```

Comment:
you can do `.zero`. also I think this is unnecessary b/c default should be `.zero` already